### ISSUE TITLE
W0611 unused-import false positive

### ISSUE THREAD

**Issue by teake** [PyCQA/pylint#3445]
This one seems related to #3444.

### Steps to reproduce

1.  Make a Python file (`w0611.py`) with the following content:
```
from os import environ

for k, v in environ.items():
    print(k, v)

```

2. Run `pylint --disable=all --enable=W0611 w0611.py`

### Current behavior

Output:
```
************* Module w0611
w0611.py:1:0: W0611: Unused environ imported from os (unused-import)

-------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 10.00/10, -3.33)
```

### Expected behavior

That W0611 isn't raised.

### pylint --version output

This is on the master version of both pylint and astroid at the time of writing.

```
pylint 2.5.0-dev1
astroid 2.4.0
Python 3.7.6 | packaged by conda-forge | (default, Mar  5 2020, 15:03:29) 
[Clang 9.0.1 ]
```

**Comment by teake** [comment]
Thanks! 💪
